### DISCUSSION TITLE
jupyter lab怎麼一直沒跑出東西

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#217] [Q&A] [answered]
我發現我的jupyter lab跑不出東西，左邊有一個*，不知道是發生什麼事。
![image](https://user-images.githubusercontent.com/86505454/147472404-ad879325-59c7-4a71-acea-9a9f406ddf27.png)

**Comment by enoch-lightning** [comment]
居然連print(2)都跑不出來。
![image](https://user-images.githubusercontent.com/86505454/147472514-97275cc6-2580-473e-92ff-525805c369f6.png)

**Comment by enoch-lightning** [comment]
剛剛先把jupyter lab視窗(頁籤)關掉，再把命令提示字元(cmd)中的
`file:///C:/Users/enoch/AppData/Roaming/jupyter/runtime/jpserver-15568-open.html`
複製(ctrl+c)下來，再貼到(ctrl+v)網址列搜尋，重新用jupyter lab打開同一個Untitled 1.ipynb，就可以動了！不知道問題出在哪裡。
![image](https://user-images.githubusercontent.com/86505454/147473084-a277be44-861a-42e6-819f-2d21b9d000d7.png)

**Comment by ghost** [comment, marked as answer]
印象中顯示*字號就是有點當掉的樣子(還在跑 有時候就當在那裡了 重開應該就可以了?
